pkg/ingress/model_build_listener_rules.go

```diff
@@ -33,7 +33,15 @@ func (t *defaultModelBuildTask) buildListenerRules(ctx context.Context, lsARN co
 				enhancedBackend, err := t.enhancedBackendBuilder.Build(ctx, ing.Ing, path.Backend,
 					WithLoadBackendServices(true, t.backendServices),
 					WithLoadAuthConfig(true))
-				if err != nil {
+				if errors.Is(err, errConditionValidation) {
+					t.logger.Error(err, "Ingress has invalid annotation for service, thus will be ignored",
+						"Ingress",
+						ing,
+						"Service name",
+						path.Backend.Service.Name,
+					)
```

Comment:
This isn't the clearest error message. And logging an entire `ClassifiedIngress` is noisy.

Perhaps something like "Ignoring Ingress _NAMESPACE_/_NAME_: alb.ingress.kubernetes.io/conditions._SERVICE_ annotation: _ERROR_"

Reply:
Sticking the same style of logging error in that project, where I noticed first go a common error message at the beggining and then entities involved in the error in key-values format, e.g.
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/a92e689dfe464f5b24784f398947e0fef31dc470/controllers/elbv2/eventhandlers/node.go#L92
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/a92e689dfe464f5b24784f398947e0fef31dc470/pkg/backend/endpoint_resolver.go#L187
I'll change your suggested error format a little, but anyway make it less noisy
```go
t.logger.Error(err, "Ignoring Ingress invalid annotation",
  "Ingress", fmt.Sprintf("%s/%s", ing.Ing.Namespace, ing.Ing.Name),
  "Service", path.Backend.Service.Name,
)
```

Reply:
Consider logging the annotation key. It might not be obvious that it's the `actions.` annotation this is referring to.

Reply:
> It might not be obvious that it's the actions.

@johngmyers Probably you've meant `conditions.`, I'll add that info to error.